content/1.2/docs/serverless-workflows/troubleshooting.md

```diff
@@ -94,3 +94,27 @@ a CA which is not available to the workflow. The error in the workflow pod log u
 
 **Solution:**
 When deploying a workflow in a namespace other than the one where Sonataflow services are running (e.g., sonataflow-infra), there are essential steps to follow to enable persistence and connectivity for the workflow. See the following [steps](https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/docs/release-1.2/README.md#additional-workflow-namespaces).
+
+### Problem: sonataflow-platform-data-index-service pods can't connect to the database on startup
+
+This can be cause by a network policy issue if you deployed RHDH Operator in a namespace different from `rhdh-operator`.
+
+**Cause:**
+Network policies may be restricting communication between pods within the `sonataflow-infra` namespace, preventing the SonataFlow platform pods from connecting to the database.
+
+**Solution:**
+Create a NetworkPolicy that allows communication between pods whitin `sonataflow-infra` namespace:
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-intra-namespace
+  namespace: sonataflow-infra
+spec:
+  podSelector: {}
+  policyTypes:
+    - Ingress
+  ingress:
```

Comment:
For the ingress part, if you want to allow communication within the same namespace, it should be
```
- Ingress
  ingress:
  - from:
    - namespaceSelector:
        matchLabels: rhdh.redhat.com/workflow-namespace: ""
```
But this is already set by https://github.com/parodos-dev/orchestrator-helm-operator/blob/main/helm-charts/orchestrator/templates/network-policies.yaml#L20